app/utils/homepageManagement.js

```diff
@@ -0,0 +1,52 @@
+import {
+  HUBSPOT_URL,
+  HUBSPOT_PORTAL_ID,
+  HUBSPOT_SEND_EMAIL_FORM_ID,
+  HUBSPOT_SEND_MESSAGE_FORM_ID,
+} from './constants';
+
+export async function sendEmail(formData, pageInfo) {
+  const fields = Object.keys(formData).map(name => ({
+    name,
+    value: formData[name],
+  }));
+
+  await window.$.ajax({
```

Comment:
Let's use 'fetch' or 'axios' library. Choose whatever you like more.
These are modern libraries that provide more capabilities and easier to use.